config/config.go

```diff
@@ -158,6 +160,8 @@ type Config struct {
 	EnableGlobalIndex bool `toml:"enable-global-index" json:"enable-global-index"`
 	// DeprecateIntegerDisplayWidth indicates whether deprecating the max display length for integer.
 	DeprecateIntegerDisplayWidth bool `toml:"deprecate-integer-display-length" json:"deprecate-integer-display-length"`
+	// EnableLogDesensitization indicates that whether desensitization when log query.
+	EnableLogDesensitization bool `toml:"enable-log-desensitization" json:"enable-log-desensitization"`
```

Comment:
I am still hoping that from a product perspective we can change from the term "desensitize" to the term "redact".

Reply:
Thanks, Already change the variable name from `tidb_log_desensitization` to `tidb_redact_log`.